.github/workflows/publish_helm_chart.yaml

```diff
@@ -0,0 +1,44 @@
+# package and push the zenml helm chart to the ECR repository
+name: Publish Helm Chart
+
+on: workflow_call
```

Comment:
Might make sense to also allow this to be called from workflow dispatch in case it fails on release